BackofficeBundle/Resources/views/Tree/tree-macros.html.twig

```diff
@@ -12,7 +12,10 @@
             >
                 <span>{{ node.name }}</span>
             </a>
-            {% if 'child' in element|keys and (element.child is not empty or is_granted("ROLE_ACCESS_CREATE_NODE"))%}
+            {% if 'child' in element|keys
+               and (element.child is not empty or is_granted("ROLE_ACCESS_CREATE_NODE"))
+               and node.nodeId != constant('TRANSVERSE_NODE_ID', node)
```

Comment:
I think that constant should take the fully qualified namespace of the class and then the constant name